### ISSUE TITLE
Multiline Profile property does not "render" breaks on user profile page

### ISSUE THREAD

**Issue by Timo-Breumelhof** [dnnsoftware/Dnn.Platform#4487]
## Description of problem
When you set a Profile property field to "Multi Line Text" and you add this to the user profile page the entered breaks are not visible.

## Description of solution
IMO we should replace "\n" with "<br>", so the entered breaks are visible when shown in the profile.
I know this is a change, but if we don't change this, what's the use of the "Multi-Line Text" type..?

As all properties seem to be handled here: https://github.com/dnnsoftware/Dnn.Platform/blob/34a5a5582c394d45e76baec656ada86d3078453c/DNN%20Platform/Library/Entities/Users/Profile/ProfilePropertyAccess.cs#L193

I would suggest to add an option for "Multi-Line Text" in there and add a .Replace("\n", "<br>").

Reproduce.
1. Set a profile property (Biography) to "Multi Line Text"

![image](https://user-images.githubusercontent.com/8999382/107758068-a9ce8180-6d26-11eb-808f-a0a3af054092.png)

2. Edit a user profile and add this text as Biography. 
```
-------------
Test1
Test2
Test3
-------------
```
3. Visit the User Profile page of that user.
4. Edit the "ViewProfile" module's Template and add: [PROFILE:BIOGRAPHY]
5. After the page has reloaded you will see that the entered breaks are not shown.
(as obviously "\n" is not a break in HTML)

![image](https://user-images.githubusercontent.com/8999382/107758796-b1425a80-6d27-11eb-91c5-0f85beee0108.png)

## Affected browser

* [ ] Chrome
* [ ] Firefox
* [ ] Safari
* [ ] Internet Explorer
* [ ] Edge

**Comment by Timo-Breumelhof** [comment]
FYI if we can agree on the solution we will create the PR.

**Comment by sleupold** [comment]
@Timo-Breumelhof 
thank you, your solution sounds great to me!

**Comment by bdukes** [comment]
Yes, either replace `\n` with `<br/>` or render using a CSS class that applies `white-space: pre-line`

**Comment by hismightiness** [comment]
I had to do something similar in another issue/PR:  https://github.com/dnnsoftware/Dnn.Platform/pull/4186

**Comment by hismightiness** [comment]
Still relevant.

**Comment by Timo-Breumelhof** [comment]
:-)

**Comment by valadas** [comment]
Is this still reproducible in v9.10.1, I have a feeling a PR came for this, can someone test/confirm...

**Comment by hismightiness** [comment]
Unfortunately, that PR that @valadas mentioned is not working for this release.  I had made that PR a few releases back.  Weird...  

![image](https://user-images.githubusercontent.com/938023/132378403-90aad663-3ee5-4da2-84f8-f18ab857169f.png)

**Comment by valadas** [comment]
Yeah I can reproduce in 9.10.1
